sdk/metric/batcher/defaultkeys/defaultkeys_test.go

```diff
@@ -95,7 +95,7 @@ func TestGroupingStateful(t *testing.T) {
 	err := checkpointSet.ForEach(records1.AddTo)
 	require.NoError(t, err)
 
-	require.EqualValues(t, map[string]int64{
+	require.EqualValues(t, map[string]float64{
```

Comment:
Why this change? I don't get it.

Reply:
(None of the batcher tests were testing float instruments, I guess that's another reason to make the change.)